x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -1255,78 +1255,78 @@ public void testEmptyEsRelationOnCountStar() throws IOException {
     }
 
     public void testUnsupportedFieldsInStats() {
-        var errorMsg = "Cannot use field [shape] with unsupported type [geo_shape]";
+        var errorMsg = "Cannot use field [unsupported] with unsupported type [ip_range]";
```

Comment:
Slightly awkward that the unsupported field is called `unsupported`, but on the other hand perhaps this field will never be supported, and this is perfect. 🤷